src/components/PageElements/Elements/HistoryResponseElement.tsx

```diff
@@ -1,23 +1,19 @@
 import React from 'react'
 import { Header, Icon } from 'semantic-ui-react'
+import { SummaryViewWrapper } from '../../../formElementPlugins'
 import getSimplifiedTimeDifference from '../../../utils/dateAndTime/getSimplifiedTimeDifference'
+import { ElementState, HistoryElement } from '../../../utils/types'
 
-interface HistoryResponseElementProps {
-  author?: string
-  title: string
-  message: string
-  timeUpdated: Date
-  // attentionRequired?: boolean
-  reviewerComment?: string
-}
```

Comment:
This interface not required, it's the same as the "HistoryElement" type defined in "types.ts"